src/widget/container.rs

```diff
@@ -10,40 +11,53 @@ pub struct Border<Color> {
     pub width: u32,
 }
 
-#[derive(Clone, Copy, Default, Eq, PartialEq)]
-pub struct Options<Color> {
-    pub background_color: Option<Color>,
-    pub border: Option<Border<Color>>,
+pub struct Options<Display>
```

Comment:
See some discussion below about changing widgets to be parameterized by `Display` instead of `Color`.